lib/models/index.ts

```diff
@@ -0,0 +1,116 @@
+import { InjectedDependenciesParam, InjectedOptionsParam } from '../balena';
+
+/*
+Copyright 2016 Balena
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const modelsTemplate = [
+	/**
```

Comment:
I converted to typescript as I could not put jsdoc comments within an array in coffeescript

Reply:
With the new object form I could technically keep it in coffeescript but I don't think it's worth reverting :P  (although we don't actually get typing benefits atm, it does at least make it easier to add proper typings in future)